### ISSUE TITLE
Negative Sampling on CP

### ISSUE THREAD

**Issue by pvcastro** [thunlp/RE-Context-or-Names#10]
Hi there!

Congratulations for your paper and for your results!

I'm missing from the dataset the point in which CP would yield negative pairs for sampling during training, to satisfy the contrastive learning objective. I only saw the code for MTB dataset / training. 

Do you mind explaining how you are doing negative sampling in the CP benchmark, giving references in the code?

Thanks!

**Comment by h-peng17** [comment]
Hi~
We use Normalized Temperature-scaled Cross Entropy Loss(Ntxent Loss) as our loss function. We don't do negative sample explicitly. Instead the negative sampling was done in Ntxentloss class. You can see https://github.com/KevinMusgrave/pytorch-metric-learning for more details about Ntxentloss.

**Comment by pvcastro** [comment]
OK, thanks! 
I'll take a look!
